src/common/__test__/config.spec.js

```diff
@@ -1,18 +1,22 @@
 import { getGlobalConfig, assembleBuildConfig, setGlobalConfig } from '../config';
 
-test('Default globalConfig properties should be all true', () => {
+const customDummyLoggerFunction = (msg => console.log(msg));
+
+test('Default globalConfig properties should be all true + console should be the logger', () => {
     expect(getGlobalConfig()).toEqual({
         url: true,
         method: true,
         data: true,
         status: true,
+        logger: console,
```

Comment:
How about `console.log`?

Reply:
with your suggestion above implemented, yes, that will be the most sensible default, I guess 👍